src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
+  box-sizing: content-box;
+  max-width: 1024px;
+}
```

Comment:
It's generally recommended to use box-sizing: border-box; because it includes padding and border widths in the element's total width and height. This can make it easier to calculate and control the dimensions of your elements, especially when working with responsive layouts.